simple-erc20-spl-hardhat/README.md

```diff
@@ -0,0 +1,114 @@
+# Example deploying ERC20ForSplMintable to Neonlabs devnet using Hardhat
+
+This directory contains all the files necessary to deploy ERC20ForSplMintable contract using Neon onto the Solana blockchain.
```

Comment:
Please add link to https://docs.neon-labs.org/docs/developing/deploy_facilities/interacting_with_spl_tokens